AppCenterData/AppCenterDataTests/MSDataTests.m

```diff
@@ -622,10 +628,319 @@ - (void)testListWhenOffline {
                                  }
                                }];
   XCTAssertNotNil(actualPaginatedDocuments);
-  XCTAssertNotNil(actualPaginatedDocuments.currentPage.error);
-  XCTAssertNotNil(actualPaginatedDocuments.currentPage.error);
-  XCTAssertEqual(actualPaginatedDocuments.currentPage.error.domain, kMSACDataErrorDomain);
-  XCTAssertEqual(actualPaginatedDocuments.currentPage.error.code, MSACDataErrorHTTPError);
+  XCTAssertNil(actualPaginatedDocuments.currentPage.error);
+  XCTAssertEqual([[actualPaginatedDocuments currentPage] items].count, 0);
+}
+
+- (void)testListReturnsFromLocalStorageWhenOffline {
+
+  // If
+  __weak XCTestExpectation *expectation = [self expectationWithDescription:@""];
```

Comment:
Better to add a description to your expectation, it'll show up in tests logs on the CI.